apps/order.rb

```diff
@@ -0,0 +1,43 @@
+RodaApp.route('order') do |r|
+
+  # POST /api/orders/new
+  r.post 'new' do
+    message = CreateOrder.new(r.params).call
+    PostToSlack.say(:food_bot, message) unless test?
+    status(200)
+  end
+
+  # POST /api/orders/show
+  r.post 'show' do
+    message = ShowOrderList.message
+    PostToSlack.say(:food_bot, message) unless test?
```

Comment:
This unless can be deleted because of https://github.com/binarapps/slack-integrator/pull/14